test/cql-pytest/README.md

```diff
@@ -30,6 +30,7 @@ Scylla.
 Additional options can be passed to "pytest" or to "run" / "run-cassandra"
 to control which tests to run:
 
+### How to run
```

Comment:
I don't think this heading is appropriate. The text above already explains the basics of how to run. This section only presents additional command line options that are useful, but is not the right place to start reading when you just want to run. So why do we need this extra heading at all?

Reply:
Done